components/notebook-controller/Dockerfile.konflux

```
@@ -10,31 +10,25 @@ ARG TARGETARCH
 ARG SOURCE_CODE
 
 WORKDIR /opt/rhods
+
 # Copy the Go Modules manifests
 COPY ${SOURCE_CODE}/notebook-controller ./notebook-controller
 
 # This is necessary because the Jupyter controller now depends on
 # components/common
 COPY ${SOURCE_CODE}/common ./common
 
-COPY ${SOURCE_CODE}/odh-notebook-controller ./odh-notebook-controller
-
-WORKDIR /opt/rhods/odh-notebook-controller
+# Update building workdir
+WORKDIR /opt/rhods/notebook-controller
 
 ## Build the odh-notebook-controller
 USER root
 
 # Build
-RUN CGO_ENABLED=1 GOOS=${TARGETOS:-linux} GOARCH=${TARGETARCH} GO111MODULE=on go build -tags strictfipsruntime -a -mod=mod -o manager main.go
+RUN  CGO_ENABLED=1 GOOS=${TARGETOS:-linux} GOARCH=${TARGETARCH} go build -tags strictfipsruntime -a -o ./bin/manager main.go
```

```suggestion
RUN CGO_ENABLED=1 GOOS=${TARGETOS:-linux} GOARCH=${TARGETARCH} go build -tags strictfipsruntime -a -o ./bin/manager main.go
```

Isn't that extra space a typo? I wouldn't sync _that_ ;p